Dockerfile

```diff
@@ -1,33 +1,29 @@
-FROM ubuntu:14.04
-
-# Add openMVG binaries to path
-ENV PATH $PATH:/opt/openMVG_Build/install/bin
-
-# Get dependencies
-RUN apt-get update && apt-get install -y \
-  build-essential \
-  cmake \
-  graphviz \
-  git \
-  gcc-4.8 \ 
-  gcc-4.8-multilib \  
-  libpng-dev \
-  libjpeg-dev \
-  libtiff-dev \
-  libxxf86vm1 \
-  libxxf86vm-dev \
-  libxi-dev \
-  libxrandr-dev \
-  python-dev \  
-  python-pip
-
-# Clone the openvMVG repo 
-ADD . /opt/openMVG
-RUN cd /opt/openMVG && git submodule update --init --recursive
-
-# Build
-RUN mkdir /opt/openMVG_Build && cd /opt/openMVG_Build && cmake -DCMAKE_BUILD_TYPE=RELEASE \
-  -DCMAKE_INSTALL_PREFIX="/opt/openMVG_Build/install" -DOpenMVG_BUILD_TESTS=ON \
-  -DOpenMVG_BUILD_EXAMPLES=ON . ../openMVG/src/ && make
-
-RUN cd /opt/openMVG_Build && make test
+ARG CUDA_TAG=9.2
+ARG OS_TAG=7
+FROM nvidia/cuda:${CUDA_TAG}-devel-centos${OS_TAG}
+LABEL maintainer="AliceVision Team alicevision-team@googlegroups.com"
+
+# Install all compilation tools
+# file: needed by cmake
+RUN yum -y install file build-essential make git wget unzip yasm pkg-config libtool nasm automake openssl-devel gcc-gfortran
+
+# Manually install cmake 3.11
+RUN cd /opt && wget https://cmake.org/files/v3.11/cmake-3.11.0.tar.gz && tar zxvf cmake-3.11.0.tar.gz && cd cmake-3.11.0 && ./bootstrap --prefix=/usr/local  -- -DCMAKE_BUILD_TYPE:STRING=Release -DCMAKE_USE_OPENSSL:BOOL=ON && make -j8 && make install
+
+ENV AV_DEV=/opt/AliceVision_git \
+    AV_BUILD=/tmp/AliceVision_build \
+    AV_INSTALL=/opt/AliceVision_install \
+    AV_BUNDLE=/opt/AliceVision_bundle \
```

Comment:
what's the difference between install and bundle?

Reply:
/opt/AliceVision_install will have the bin, include, lib64, share folders with all the built dependencies.
/opt/AliceVision_bundle will have all libs and binaries together in the same folder without include, share, but use the bundle mechanism to retrieve all used libraries, including the ones that comes from the OS like cuda libraries (as they are installed on the OS level by the base docker image), libspqr, libcamd, libamd, etc.